### PR TITLE
feat: use functions-types as source of type definitions

### PR DESCRIPTION
**Pull request by msieroslawska**
We have created type definitions for NER events in `functions-types` package, and now we want to consolidate all NER-related types to be imported from that one source of truth. This PR removes the dependency to `node-apps-toolkit` (it will also be pulling type definitions from `functions-types`).

### CODE REVIEW

File: examples/native-external-references/functions/helpers.spec.ts [hunk 1]
```diff
@@ -51,7 +51,9 @@ describe('Getting the URLs', () => {
       lookupUrls: [
         'https://api.themoviedb.org/3/movie/urn:123?language=en-US',
         'https://api.themoviedb.org/3/movie/urn:456?language=en-US'
-      ]
+      ],
+      trendingUrl:
+        'https://api.themoviedb.org/3/trending/movie/day?language=en-US'
```
Comment (screamcha) [inline examples/native-external-references/functions/helpers.spec.ts:56]: Why is this addition needed?

Comment (msieroslawska) [reply examples/native-external-references/functions/helpers.spec.ts:56]: By default first search query is empty and TMDB responds with an empty array. To have _something_ displayed at first render of entity picker (before user starts typing the query), we've decided to pull results from the trending endpoint. We forgot to cover this in the example tests.

**Review by screamcha — APPROVED** [review]
